engines/bastion_katello/test/content-hosts/content/content-host-packages-installed.controller.test.js

```diff
@@ -66,7 +66,7 @@ describe('Controller: ContentHostPackagesInstalledController', function() {
 
         spyOn(HostPackage, 'remove');
         $scope.removeSelectedPackages();
-        expect(HostPackage.remove).toHaveBeenCalledWith({id: mockHost.id, packages: [mockPackageClone]},
+        expect(HostPackage.remove).toHaveBeenCalledWith({id: mockHost.id, packages: ['foo']},
```

Comment:
the `mockPackageClone` here contains various NVRA information in different fields which are not in the table today - which has whole `nvra`, `nevra` attributes - I'm not sure if the package table held data in that format before, but now the test matches the behavior I think we really intend.